README.md

```diff
@@ -109,5 +86,4 @@ of the discussion board!
 
 ## License - Much license ⚖️
 Dogecoin Core is released under the terms of the MIT license. See
-[COPYING](COPYING) for more information or see
-[opensource.org](https://opensource.org/licenses/MIT)
+[COPYING](COPYING) for more information.
```

Comment:
Removing this link is definitely within the letter of this PR, but I think it's the least likely URL to be hijacked in a way that targets Dogecoin in specific. I have no objection to this change in principle though.

Reply:
Since there's nothing on the linked page that we don't list in `COPYING`, I found it not worth making an exception for it. Instead I minimized to have only the GitHub repo exception, to not create a completely unworkable situation.